steps/containers_kube.py

```diff
@@ -9,6 +9,13 @@ def step_impl(context, image, host):
                                host,
                                module_args='docker pull %s' % image)
 
+@then('docker pull "{image}"')
+def step_impl(context, image):
+    '''docker pull image'''
+    assert context.remote_cmd('command',
+                               remote_user='root',
```

Comment:
Does `remote_user` actually work yet?  I think we need to fix up the dynamic inventory script first.